libs/chart/src/lib/trading-chart.tsx

```diff
@@ -18,15 +48,140 @@ export const TradingChartContainer = ({
 
   return (
-    <TradingChart
-      dataSource={dataSource}
-      interval={Interval.I15M}
-      theme={theme}
-    />
+    <div className="h-full flex flex-col">
+      <div className="flex flex-row flex-wrap gap-8">
+        <div>
+          {Object.values(Interval).map((timeInterval) => (
+            <Button
+              variant="inline"
+              onClick={() => {
+                setInterval(timeInterval);
+              }}
+            >
+              <span
+                className={classNames({
+                  'font-bold': timeInterval === interval,
+                })}
+              >
+                {timeInterval}
+              </span>
+            </Button>
+          ))}
+        </div>
+        <DropdownMenu>
+          <DropdownMenuTrigger>
+            <Button appendIconName="caret-down" variant="secondary">
+              <Icon name={chartTypeIcon.get(chartType) as IconName} />
+            </Button>
+          </DropdownMenuTrigger>
+          <DropdownMenuContent>
+            <DropdownMenuRadioGroup
+              value={chartType}
+              onValueChange={(value) => {
+                setChartType(value as ChartType);
+              }}
+            >
+              {Object.values(ChartType).map((type) => (
+                <DropdownMenuRadioItem key={type} inset value={type}>
+                  <DropdownMenuItemIndicator>
+                    <Icon name="tick" />
+                  </DropdownMenuItemIndicator>
+                  {chartTypeLabels[type]}
+                </DropdownMenuRadioItem>
+              ))}
+            </DropdownMenuRadioGroup>
+          </DropdownMenuContent>
+        </DropdownMenu>
+        <DropdownMenu>
+          <DropdownMenuTrigger>
+            <Button appendIconName="caret-down" variant="secondary">
+              Overlays
+            </Button>
+          </DropdownMenuTrigger>
+          <DropdownMenuContent>
+            {Object.values(Overlay).map((overlay) => (
+              <DropdownMenuCheckboxItem
+                key={overlay}
+                checked={overlays.includes(overlay)}
+                inset
+                onCheckedChange={(checked) => {
+                  const newOverlays = [...overlays];
+
+                  const index = overlays.findIndex((item) => item === overlay);
+
+                  index !== -1
+                    ? newOverlays.splice(index, 1)
+                    : newOverlays.push(overlay);
+
+                  setOverlays(newOverlays);
+                }}
+              >
+                <DropdownMenuItemIndicator>
+                  <Icon name="tick" />
+                </DropdownMenuItemIndicator>
+                {overlayLabels[overlay]}
+              </DropdownMenuCheckboxItem>
+            ))}
+          </DropdownMenuContent>
+        </DropdownMenu>
+        <DropdownMenu>
+          <DropdownMenuTrigger>
+            <Button appendIconName="caret-down" variant="secondary">
+              Studies
```

Comment:
@mattrussell36 / @bglownia do we have i18n stuff in yet or is this how we're going for now?

Reply:
We have the `t` function. All it does is return the string you pass in, we are just using it to make it easier to swap out for a proper translation layer later on
```suggestion
{t('Studies')}
```